.github/ISSUE_TEMPLATE/bounty.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bounty Template
+about: Create a bounty
+title: ''
+labels: 'bounty, new issue'
+assignees: ''
+---
+
+<!--- Please DO NOT remove the automatically added 'new issue' label -->
+<!--- Provide a general summary of the issue in the Title above -->
+
+### Job Story
+
+### Description
+
+### Definition of Done
+
+### Developing HOPR
+
+- [running a local environment](https://github.com/hoprnet/hoprnet#develop)
+
+### What is HOPR?
+
+Find out more about HOPR at the following links:
```

Comment:
If there's going to be a bounty channel in Discord, does it make sense to link that here? This feels the most suitable place in this current template, but it doesn't exactly fit "What is HOPR?"

Or perhaps the link should be given in its own section (or in a different place entirely)

Reply:
I would like to link discord here, will add.